App.js

```diff
@@ -77,6 +83,42 @@ class App extends Component {
     LayoutAnimation.spring();
   }
 
+  getCurrentRouteName(navigationState) {
+    if (!navigationState) {
+      return null;
+    }
+    const route = navigationState.routes[navigationState.index];
+
+    if (route.routes) {
+      return this.getCurrentRouteName(route);
+    }
+
+    return route.routeName;
+  }
+
+  statusBarHandler(prev, next) {
+    const routeName = this.getCurrentRouteName(next);
+    const lightContentScreens = [
+      'MyProfile',
+      'Profile',
+      'Organization',
+      'Repository',
+      'Login',
+    ];
+
+    if (lightContentScreens.includes(routeName)) {
+      StatusBar.setTranslucent(true);
+      StatusBar.setBackgroundColor(
+        routeName === 'Login' ? colors.transparent : colors.black
+      );
+      StatusBar.setBarStyle('light-content');
+    } else {
+      StatusBar.setTranslucent(false);
+      StatusBar.setBackgroundColor(colors.grey);
```

Comment:
colors.white look nicer to me (give the same transparent look on Android)

Reply:
This transparent look is not a "default" look and feel of material design ( as described [here](https://developer.android.com/training/material/theme.html?hl=pt-br) ). What do you think about @machour? Should we keep the material design thing or it is better to have the same iOS design? 😄

Reply:
I completely forgot about material design 😅 
Let's keep it like you did it then